Dockerfile

```diff
@@ -102,6 +102,8 @@ RUN echo ${ILIOS_VERSION} > VERSION
 
 COPY docker/fpm/symfony.prod.ini $PHP_INI_DIR/conf.d/symfony.ini
 COPY docker/fpm/ilios.ini $PHP_INI_DIR/conf.d/ilios.ini
+COPY docker/fpm/xdebug.ini $PHP_INI_DIR/conf.d/xdebug.ini
```

Comment:
This should also go into the dev container. Probably name it `xdebug-dev.ini` or something for clarity.

Reply:
you got it. :+1: